app/bundles/CampaignBundle/Entity/LeadEventLogRepository.php

```diff
@@ -196,10 +196,6 @@ public function getUpcomingEvents(array $options = null)
 
         $query->orderBy('ll.trigger_date');
 
-        if (!empty($ipIds)) {
```

Comment:
I can't find a reference to `$ipIds` anywhere in Mautic, so this change seems good.

Reply:
Good to be removed 👍